registry/proxy/proxyblobstore.go

```diff
@@ -50,6 +50,11 @@ func (pbs *proxyBlobStore) copyContent(ctx context.Context, dgst digest.Digest,
 		setResponseHeaders(w, desc.Size, desc.MediaType, dgst)
 	}
 
+	if !copydata {
```

Comment:
This is an odd interface to have a function called "copy" not actually copy if a flag is set.

Please fix it so this check is made before entering this method.